builtin/index-pack.c

```diff
@@ -481,7 +481,7 @@ static void *unpack_raw_entry(struct object_entry *obj,
 	unsigned char *p;
 	size_t size, c;
 	off_t base_offset;
-	unsigned shift;
+	size_t shift;
```

Comment:
Nicely done, but let's not state that this is for Windows. This is a data type cleanup that benefits 64-bit systems where `long` is 32 bit wide.

Reply:
OK